### PR TITLE
add docs for EMCPy

### PR DESCRIPTION
**Pull request by kevindougherty-noaa**
## Description

Add markdown files for EMCPy backend documentation.

### CODE REVIEW

File: doc/eva_user_guide/plotting_backends/EMCPy/batch_plotting.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+# Batch Plotting and Using EMCPy Backend for eva
+
+In addition to the `datasets` and `transforms` section within the configuration YAMLs used within eva, there is a `graphics` section. This section is used to design the figures needed during analysis of your data using EMCPy's plotting backend. 
+
+An example of a `graphics` section from a test configuration file for both a scatter plot and map scatter plot is shown below:
+
+```
```
Comment (CoryMartin-NOAA) [inline doc/eva_user_guide/plotting_backends/EMCPy/batch_plotting.md]:
```suggestion
``` yaml
```

File: doc/eva_user_guide/plotting_backends/EMCPy/batch_plotting.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+          vmin: ${dynamic_vmin}
+          vmax: ${dynamic_vmax}
+            
+```
+
+## Batch Figure
+
+Given the variables selected in the `datasets` section of the configuration yaml, the user can create batch figures for all variables selected. In this example, a scatter plot and map plot will be created for all the variables the user selected to be analyzed.
+
+```
```
Comment (CoryMartin-NOAA) [inline doc/eva_user_guide/plotting_backends/EMCPy/batch_plotting.md]:
```suggestion
``` yaml
```

File: doc/eva_user_guide/plotting_backends/EMCPy/batch_plotting.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+  - batch figure:
+      variables: *variables
+```
+
+If the dataset was to use radiance data, the user could indicate specific channels and include them in the batch figure section to create plots for all variables and channels selected.
```
Comment (CoryMartin-NOAA) [inline doc/eva_user_guide/plotting_backends/EMCPy/batch_plotting.md]:
```suggestion
If the dataset included radiance data, the user could indicate specific channels and include them in the batch figure section to create plots for all variables and channels selected.
```

File: doc/eva_user_guide/plotting_backends/EMCPy/batch_plotting.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+EMCPy's declarative plotting design is separated into three sections: 
+ * figure
+ * plots
+ * layers
+
+Each section is responsible for different parts of the figure design which are explained below.
+
+### Figure
+
+This section is responsible for the high-end figure design such as the layout of subplots, the figure title, figure size, output filename and more.
```
Comment (CoryMartin-NOAA) [inline doc/eva_user_guide/plotting_backends/EMCPy/batch_plotting.md]:
```suggestion
This section is responsible for the high-level figure design such as the layout of subplots, the figure title, figure size, output filename and more. 
```

**Review by CoryMartin-NOAA — COMMENTED** [review]
Couple of comments. Mainly: be explicit with `yaml` before the code snippets for ensuring proper formatting.

File: doc/eva_user_guide/plotting_backends/EMCPy/batch_plotting.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+          colorbar: true
+          cmap: ${dynamic_cmap}
+          vmin: ${dynamic_vmin}
+          vmax: ${dynamic_vmax}
+            
+```
+
+## Batch Figure
+
+Given the variables selected in the `datasets` section of the configuration yaml, the user can create batch figures for all variables selected. In this example, a scatter plot and map plot will be created for all the variables the user selected to be analyzed.
```
Comment (danholdaway) [inline doc/eva_user_guide/plotting_backends/EMCPy/batch_plotting.md]:
```suggestion
Given the variables selected in the `datasets` section of the configuration YAML, the user can create batch figures for all variables selected. In this example, a scatter plot and map plot will be created for all the variables the user selected to be analyzed.
```
I think we've been capitalizing YAML most places including elsewhere on this page.

File: doc/eva_user_guide/plotting_backends/EMCPy/batch_plotting.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+          x:
+            variable: experiment::ObsValue::${variable}
+          y:
+            variable: experiment::hofxPassedQc::${variable}
+          markersize: 5
+          color: 'red'
+          label: 'JEDI h(x) versus obs (passed QC in JEDI)'
+```
+
+The above example is plotting two layers of scatter plot data on a single subplot. The first layer compares experiment::ObsValue::${variable} as the x values and variable: experiment::hofx::${variable} as the y values using black dots and adding the appropriate label. The second layer uses different data in red and adds a label as well.
```
Comment (danholdaway) [inline doc/eva_user_guide/plotting_backends/EMCPy/batch_plotting.md]:
```suggestion
The above example is plotting two layers of scatter plot data on a single subplot. The first layer compares `experiment::ObsValue::${variable}` as the x values and variable: `experiment::hofx::${variable}` as the y values using black dots and adding the appropriate label. The second layer uses different data in red and adds a label as well.
```

**Review by danholdaway — APPROVED** [review]
Two minor comments. Thanks for adding this.

**Pull request merged**